src/types/index.js

```diff
@@ -1,2 +1,3 @@
 export { default as keyValue } from './keyValue/index'
-export { default as simpleObject } from './simpleObject/index'
+export { default as simple } from './simple/index'
+export { default as simpleObject } from './simple/index'
```

Comment:
You export it but don't use it in the factory file (src/index.js)

Reply:
yes, but it's a simple way to keep `simpleObject`.
it's use at this line : https://github.com/alakarteio/k-redux-factory/pull/119/files#diff-1fdf421c05c1140f6d71444ea2b27638R28

Reply:
OK I see 👍